README.md

```diff
@@ -60,9 +60,45 @@ the squashed commit also according to the convention.
 
 ## Translations
 
-- Check out [react-i18next](https://react.i18next.com/) and [i18next](https://www.i18next.com/) docs for more info.
+- Check out [nitrolib](https://github.com/kiwicom/nitrolib) docs for more info.
 - [DOCs for Phraseapp](https://phraseapp.com/docs/) - where the translations are stored for translators
-- List of all supported languages is defined by file `i18n/languages.js`
+- List of all supported languages is defined by file `src/translations/langInfos.js`
+
+### How to use translations in code
+
+- Use `Text` from [nitrolib](https://github.com/kiwicom/nitrolib) as `Trans` and don't confuse it with `Text` from [Orbit](https://github.com/kiwicom/orbit-components):
```

Comment:
wow, nitro and orbit should co-operate to merge this into one component with optionally disable-able translations :-D